src/lib/components/blog/blog-entry.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+  import { makeBlogPostPageUrl } from '$lib/public/blog/urls';
+  import { readingTime } from '$lib/public/blog/utils';
+  import { Temporal } from '@js-temporal/polyfill';
+  import Labels from '../../../routes/blog/labels.svelte';
```

Comment:
As you can see, since Labels is under the `/routes/` directory, you get this unwanted relative path with lots of `../`s. Since the BlogPostPreview component is only expected to be used on the `/blog` page endpoint, it might make sense for the location of this file to be adjacent to the Labels component file.

Reply:
addressed